x/msgfee/antispam_query_test.go

```diff
@@ -0,0 +1,38 @@
+package msgfee
+
+import (
+	"testing"
+
+	"github.com/iov-one/weave/coin"
+)
+
+func TestAntiSpamQuery(t *testing.T) {
+	initialCoin := coin.Coin{
+		Whole:      0,
+		Fractional: 0,
+		Ticker:     "DOGE",
+	}
+
+	q := NewAntiSpamQuery(initialCoin)
+	model, err := q.Query(nil, "", nil)
+	if err != nil {
```

Comment:
You can use `assert.NoNil(err)` for this same effect (generally for all the fatal nil or error checks, easier to use it)
Import from weavetest/assert